### PR TITLE
shim: Ship BOOT.CSV in /usr/lib/shim

### PR DESCRIPTION
**Pull request by carlocaione**
https://phabricator.endlessm.com/T14870

### CODE REVIEW

**Comment by dsd** [comment]
Please explain this a bit more in the commit mesage. Why can't this script access it from its real place in the boot partition? Now that you are duplicating it in /usr how are the two copies going to be managed, synchronized (or not)?

**Comment by carlocaione** [comment]
> Why can't this script access it from its real place in the boot partition?

AFAICT `BOOT.CSV` is not in the boot partition. The only place where it is currently shipped is in the ESP and put it there at image creation time. To have the possibility to upgrade it from an ostree running system my idea was to put it into `/usr/lib/shim` so that when we have a new ostree deployment with a new SHIM shipped, we can use the copy in there to overwrite the one in the ESP.

> Now that you are duplicating it in /usr how are the two copies going to be managed, synchronized (or not)?

The script (almost done) simply mount the ESP, check if in `/usr/lib/shim` and `/usr/lib/grub` there is anything new, signed, etc... and copy the new GRUB / SHIM over the ones in the ESP.

**Comment by wjt** [comment]
Seconding what Daniel said: this rationale should be in the commit message.

File: debian/shim.install [hunk 1]
```diff
@@ -1,3 +1,4 @@
 shim.efi /usr/lib/shim
```
Comment (wjt) [inline debian/shim.install]: Is this the Microsoft-signed copy?

**Pull request closed without merging**